tools/IoTEdgeDevOps/TestDashboard/dashboards/OldEETestRunDetails.json

```diff
@@ -0,0 +1,676 @@
+            "value": "1768371"
+          },
+          {
+            "selected": false,
+            "text": "739646",
+            "value": "739646"
+          },
+          {
+            "selected": false,
+            "text": "1783121",
+            "value": "1783121"
+          },
+          {
+            "selected": false,
+            "text": "750409",
+            "value": "750409"
+          },
+          {
+            "selected": false,
+            "text": "841054",
+            "value": "841054"
+          },
+          {
+            "selected": false,
+            "text": "1768079",
+            "value": "1768079"
+          },
+          {
+            "selected": false,
+            "text": "718020",
+            "value": "718020"
+          },
+          {
+            "selected": false,
+            "text": "1631935",
+            "value": "1631935"
+          },
+          {
+            "selected": false,
+            "text": "1635207",
+            "value": "1635207"
+          },
+          {
+            "selected": false,
+            "text": "630094",
+            "value": "630094"
+          },
+          {
+            "selected": false,
+            "text": "1774016",
+            "value": "1774016"
+          },
+          {
+            "selected": false,
+            "text": "1778238",
+            "value": "1778238"
+          },
+          {
+            "selected": false,
+            "text": "1631683",
+            "value": "1631683"
+          },
+          {
+            "selected": false,
+            "text": "738641",
+            "value": "738641"
+          },
+          {
+            "selected": false,
+            "text": "725342",
+            "value": "725342"
+          },
+          {
+            "selected": false,
+            "text": "1716944",
+            "value": "1716944"
+          },
+          {
+            "selected": false,
+            "text": "739251",
+            "value": "739251"
+          },
+          {
+            "selected": false,
+            "text": "1779200",
+            "value": "1779200"
+          },
+          {
+            "selected": false,
+            "text": "870414",
+            "value": "870414"
+          },
+          {
+            "selected": false,
+            "text": "767234",
+            "value": "767234"
+          }
+        ],
+        "query": "SELECT ReleaseId FROM VstsReleaseEnvironment GROUP BY ReleaseId",
+        "refresh": 0,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "tagValuesQuery": "",
+        "tags": [],
+        "tagsQuery": "",
+        "type": "query",
+        "useTags": false
+      }
+    ]
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {
+    "refresh_intervals": [
+      "5s",
+      "10s",
+      "30s",
+      "1m",
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ]
+  },
+  "timezone": "",
```

Comment:
should it be empty?

Reply:
Definitely seems suspicious, but I don't see how it could cause problems